dev/MRTCore/packaging/native/Microsoft.ApplicationModel.Resources.WinRt.props

```diff
@@ -3,6 +3,9 @@
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
+  <!-- An empty '$(ApplicationType)' corresponds to a C++, Desktop app. -->
```

Comment:
>Desktop [](start = 59, length = 7)

do we really want to use "Desktop"? I hope we don't type app type to platform.

Reply:
using win32 as agreed upon offline